module/data/item/templates/mountable.mjs

```diff
@@ -30,7 +30,11 @@ export default class MountableTemplate extends SystemDataModel {
         max: new NumberField({ required: true, integer: true, min: 0, label: "DND5E.HitPointsMax" }),
         dt: new NumberField({ required: true, integer: true, min: 0, label: "DND5E.DamageThreshold" }),
         conditions: new StringField({required: true, label: "DND5E.HealthConditions"})
-      }, {label: "DND5E.HitPoints"})
+      }, {label: "DND5E.HitPoints"}),
+      speed: new SchemaField({
+        value: new NumberField({required: true, min: 0, label: "DND5E.Speed"}),
+        conditions: new StringField({required: true, label: "DND5E.SpeedConditions"})
+      }, {label: "DND5E.Speed"})
```

Comment:
Since these are only exposed for equipment items, does it make sense to include them with all mountable equipment?

Reply:
Maybe `required: false, initial: undefined` on the `SchemaField` so it isn't included by default in the data saved to the DB?

Reply:
I've made that change, let's try it out. If it doesn't work we can just move it back to `EquipmentData`, it's not a big deal.